### ISSUE TITLE
invalid sequence

### ISSUE THREAD

**Issue by bukowa** [linux-china/jetbrains-just-plugin#29]
This is a proper justfile (at least it evaluates correctly):

```justfile
export PATH_WEBDRIVER_BINARY := env('PATH_WEBDRIVER_BINARY',\
    if os() == 'windows' \
    {PATH_BUILD_BIN + "/msedgedriver" + exe } \
    else { shell('which $1 || true', 'WebKitWebDriver')})
```

But plugins says its invalid, the solution is to add a space after `env('PATH_WEBDRIVER_BINARY',` 

```justfile
export PATH_WEBDRIVER_BINARY := env('PATH_WEBDRIVER_BINARY', \
    if os() == 'windows' \
    {PATH_BUILD_BIN + "/msedgedriver" + exe } \
    else { shell('which $1 || true', 'WebKitWebDriver')})
```

**Comment by linux-china** [comment]
Now it's some hard to add conditionals for export. Could you use variable instead? 

```
v1 := if os() == 'windows' {  PATH_BUILD_BIN + "/msedgedriver" + exe } else { shell('which $1 || true', 'WebKitWebDriver')}

export PATH_WEBDRIVER_BINARY := env('PATH_WEBDRIVER_BINARY',v1)
```

This feature will be added in version 0.5.2.

**Comment by linux-china** [comment]
Fixed with version 0.5.2

**Comment by bukowa** [comment]
thank you
